src/components/atoms/IncompatibleModCard.astro

```diff
@@ -0,0 +1,69 @@
+---
+import { marked } from "marked";
+
+export interface Mod {
+    name: string;
+    ids: string[];
+    icon: string;
+    note?: string;
+    tracking: string;
+    status: string;
+}
+
+export interface Props {
+    mod: Mod;
+}
+
+const mod = Astro.props.mod;
+
+const [statusText, statusIcon] = {
+    "UNKNOWN": ["Status is currently unknown", "fa-question"],
+    "WONT_FIX": ["Will not be fixed", "fa-ban"],
+    "IN_PROGRESS": ["Fix in progress", "fa-spinner"],
+    "BLOCKED": ["Fix is blocked by an external factor", "fa-shield-halved"],
+    "NO_ANSWER": ["The tracking issue is awaiting a response from maintainers", "fa-hourglass-half"],
+    "ON_HOLD": ["Fix is currently on hold", "fa-pause"],
+}[mod.status || "UNKNOWN"]
```

Comment:
Feels kinda weird to have the dict inlined here rather than existing as its own constant, but that's probably just me

Reply:
Also that `||` could be a nullish coalescing operator

Reply:
Had to Google that one. Sure, let's change it.